views/lib/csrf_content.hbs

```diff
@@ -1,8 +1,8 @@
 <div class="error-header-wrapper pb-3">
-    {{> 'lib/components/svg_error_content' }}
-    <h1 class="h2 error-header-code">CSRF</h1>
+    {{> 'lib/components/svg_error_to_much_tabs' }}
+    {{!-- <h1 class="h2 error-header-code">CSRF</h1> --}}
```

Comment:
if this line is no longer needed, feel free to delete it.
In case we want to bring it back we can always find it in the git history :P